src/applications/travel-pay/components/Breadcrumbs.jsx

```diff
@@ -12,12 +12,12 @@ export default function BreadCrumbs() {
           label: 'VA.gov home',
         },
         {
-          href: '/health-care',
-          label: 'Health Care',
+          href: '/my-health',
+          label: 'My HealtheVet',
         },
         {
           href: '/health-care/file-travel-claim',
```

Comment:
If the above has changed to `/my-health`, should this line change to `href: '/my-health/file-travel-claim`?

Reply:
Semantically, you're probably right. Given the way routing is right now with the app registry, we'd have to make any changes to this value there as well. I'd say incorporate that with all the other routing updates when we get a base URL nailed down

Reply:
Can you add this item to a new or existing ticket? If we're not doing it now, I want to make sure we're doing it in conjunction with something else.

Reply:
[Created](https://app.zenhub.com/workspaces/beneficiary-travel-btsss-65147e21930cd900223d8e64/issues/gh/department-of-veterans-affairs/va.gov-team/81927)!